src/app/public/src/modules/shared/testing/test-utility.ts

```diff
@@ -1,6 +1,9 @@
+import {
+  SkyAppTestUtility
+} from '@skyux-sdk/testing/test-utility/test-utility';
+
 export class TestUtility {
   public static triggerDomEvent(element: any, eventName: string) {
-    let event = new Event(eventName);
-    element.dispatchEvent(event);
+    return SkyAppTestUtility.fireDomEvent(element, eventName);
```

Comment:
I kept this file intact just so we wouldn't need to change all the references to it in the other specs. Wouldn't hurt to change it out officially in the future.